README.md

```diff
@@ -5,106 +5,66 @@ https://sojustrepairit.org/
 The goal of the SJRK is to help youth movements and social justice initiatives to become welcoming environments for youth with learning differences, and benefit from the advantages of inclusive design. The resources available in the SJRK are intended to be lightweight and easy to deploy, share, repurpose and reuse. The kit will also be openly available to any group or individual hosting youth movements, youth action events and social justice movements.
 
 ## The Storytelling Tool
-Overall documentation for this project is available at:
+More information on this project is available at:
 https://wiki.fluidproject.org/display/fluid/Web+Storytelling+Tools
 
 This repository represents the Storytelling Tool, a project which anyone can use to share their story with the world in a way that is inclusive, multimodal and accessible. The project primarily uses [Fluid Infusion](https://fluidproject.org/infusion.html), so it is assumed that anyone working on it will have some familiarity with Infusion's syntax and philosophy.
 
 This project also uses [Node.js](https://nodejs.org), [Kettle](https://github.com/fluid-project/kettle) and [CouchDB](http://couchdb.apache.org/) to build the web-hosting environment that drives the tool.
 
-## How Do I Use It?
+## How do I use it?
 
 ### Development
+The tool is split up into two main pieces: a webserver to manage communication with the CouchDB database, and an HTML interface for creating and sharing stories.
+* Server code can be found in `src/server`
+* Common JavaScript code can be found in `src/ui`, though a significant portion of the code may be present in the theme folders (see [Theme Customization](#Theme-Customization)).
+
+## Configuring the application
+While Infusion allows for just about any possible extension you can imagine and implement, there are a few handy configuration settings provided as part of the code which allow you, for instance, to customize the current theme or configure a new one. The server will load a file named `sjrk.storyTelling.server.config.json5`, and the `globalConfig` collection is where you will find the following settings:
+
+| Setting | Description |
+|---------|-------------|
+| `port`  | _(Optional)_ The HTTP port the server will be hosted on. The default value is 8081. |
+| `savingEnabled` | Specifies whether editing and saving are allowed. If the value is set to `false`, then the site is effectively in "read-only" mode and will behave as a collection of stories rather than an authoring tool. |
+| `theme` | _(Optional)_ The theme to load the site with. If this isn't specified, a base theme will be loaded. More on this in [Theme Customization](#Theme-Customization). |
+| `themeIndexFile` | _(Optional)_ The file to serve at the site root. E.g. `"themeIndexFile": "index2.html"`. The default is `storyBrowse.html` |
```

Comment:
The statement of "The default is `storyBrowse.html`" seems inaccurate. My understanding by looking at the code is:
1. when the "base" theme is used, the value of "themeIndexFile" will be ignored and be automatically set to storyBrowse.html
2. when a custom theme is used, the value of "themeIndexFile" must be provided.

Please correct me if I'm wrong. If there are such details, better to describe them to avoid confusion. Thanks.

Reply:
The case you've described in `1.` is accurate; if `theme` isn't specified, it will load `storyBrowse.html` regardless of whether `themeIndexFile` is specified. I wonder if it makes sense to have `themeIndexFile` control the root file that gets served in all cases.
As for `2.`, the site will work if `theme` is specified and `themeIndexFile` is not. In this case, it will load the `baseTheme` index file (which is `storyBrowse.html`) but with the custom theme applied.

Reply:
Would it be better to add a note describing how `themeIndexFile` will be ignored if `theme` is not present and file a Jira for the improvement, or should I try to make it apply regardless of whether a custom theme is loaded?

Reply:
Thanks for the clarification. Yes, it would be better to describe how `themeIndexFile` works as well as the fact that the "base" theme is the fall back theme when `theme` is not specified.

Can you elaborate on "should I try to make it apply regardless of whether a custom theme is loaded"? 

Thanks.

Reply:
Sure :) what I meant by "apply regardless of whether a custom theme is loaded" is to implement the `themeIndexFile` (maybe renaming it to simply `indexFile` or `rootFile` or `defaultDocument`) in such a way that it specifies the file to serve at the site root regardless of what theme is used. I.e. if the developer wants to use `base` but change the default file from `storyBrowse.html` to, say, `storyEdit.html`, they can use the same key in `globalConfig` instead of altering `serverSetup.js`

Reply:
This change doesn't seem necessary at least for now. Let's stick with `themeIndexFile` and have a clear document for it. Thanks.